datasync/chaindatafetcher/kas/repository_checkpoint.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package kas
+
+import (
+	"errors"
+	"github.com/jinzhu/gorm"
+)
+
+const checkpointKey = "checkpoint"
+
+func (r *repository) SetCheckpoint(checkpoint int64) error {
+	data := &FetcherMetadata{
+		Key:   checkpointKey,
+		Value: checkpoint,
+	}
+
+	return r.db.Save(data).Error
+}
+
+func (r *repository) GetCheckpoint() (int64, error) {
+	checkpoint, err := r.getCheckpoint()
+	if r.isRecordNotFoundError(err) {
+		return 0, nil
+	}
+	return checkpoint, err
+}
+
+func (r *repository) getCheckpoint() (int64, error) {
```

Comment:
How about using other verbs emphasizing that functions are related with Database such as `Write/Read`, 'Store/Load`, or others.

Reply:
Applied. d7b1bf95cc14a02639c9fe75108baca9e4fe8190